### ISSUE TITLE
Clock icon missing

### ISSUE THREAD

**Issue by mikeglaz** [Bialogs/bootstrap4-datetime-picker-rails#11]
The calendar icons shows up but the clock icon is not displaying...

I am using rails 5.2.3 and font-awesome 5.11.1

I am also using the font-awesome-sass gem.

**Comment by Bialogs** [comment]
Have you configured all the icons?

Here is an example
```
$(document).on 'turbolinks:load', ->
  $('#booking-availability').datetimepicker
    icons:
      time: 'icon-clock'
      date: 'icon-calendar'
      up: 'icon-chevron-up'
      down: 'icon-chevron-down'
      previous: 'icon-chevron-left'
      next: 'icon-chevron-right'
      today: 'icon-direction'
      clear: 'icon-trash'
      close: 'icon-cross'
```

**Comment by mikeglaz** [comment]
Well, I fixed it by going down to font-awesome-sass 4.6.2...
